packages/bpk-scrim-utils/src/withScrimmedPortal.tsx

```diff
@@ -17,56 +17,51 @@
  */
 
 import type { ComponentType} from 'react';
-import { useEffect, useRef } from 'react';
+import { useState, useEffect, useRef } from 'react';
 import { createPortal } from 'react-dom';
 
 import withScrim from './withScrim';
 import type { Props as ScrimProps } from './withScrim';
 
 export type Props = ScrimProps & {
     renderTarget?: (() => HTMLElement | null) | null;
+    isPortalReady?: boolean;
 };
 
 const getPortalElement = (target: (() => HTMLElement | null) | null | undefined) => {
     const portalElement = target && typeof target === 'function' ? target() : null;
-    
+
     if (portalElement) {
         return portalElement;
     }
-    
+
     if (document.body) {
```

Comment:
I guess we don't need this bit anymore.. or we can restrucutre the code a little bit as the default will be `document.body` anyways?

Reply:
this one makes sense I think, it makes very clear the purpose of the `getPortalElement` method

Reply:
but we also set it in line 45 to `document.body`

Reply:
so maybe we can call `getPortalElement` in L45 to set the default value?

Reply:
adjusted now